src/server/replica.cc

```diff
@@ -140,12 +144,13 @@ void Replica::Stop() {
     state_mask_.store(0);  // Specifically ~R_ENABLED.
   });
 
-  waker_.notifyAll();
+  replica_waker_.notifyAll();
```

Comment:
should we notify all shard_replica_wakers here?

Reply:
I think it's not really needed, the context is cancelled above and the intervals the shard fibers awake are significantly less than 1second

Reply:
ABSL_FLAG(int, replication_acks_interval, 3000, "Interval between acks in milliseconds.");
the default is 3 seconds interval between acks in dragonfly meaninig we will wake up after 3 seconds if there is no traffic coming from master

Reply:
Ok so why do you need to call replica_waker_.notifyAll() here?
in this error handler you add a call to replica_waker_.notifyAll() as well

Reply:
fixed